plan/typeinferer.go

```diff
@@ -412,6 +412,8 @@ func (v *typeInferrer) handleFuncCallExpr(x *ast.FuncCallExpr) {
 		tp.Flen = 40
 	case ast.Coalesce:
 		tp = aggArgsType(x.Args)
+	case ast.Ord:
```

Comment:
@zimulala Hi! I just move this case statement to another place, please review it.